core/conversion/converters/impl/batch_norm.cpp

```diff
@@ -40,6 +40,7 @@ auto batch_norm_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns().
       LOG_DEBUG("momentum disregarded");
       LOG_DEBUG("training disregarded");
       LOG_DEBUG("cudnn disregarded");
+      TRTORCH_CHECK(orig_shape.nbDims > 2 , "Unable to create batch normalization layer from node: " << *n);
```

Comment:
In the current way, TRTORCH_CHECK just prints orig_shape.nbDims > 2 failed. Unable to create ... 
 Just to be more explicit, can we change these messages from "Unable to ..." to "Batch norm layer requires the input to be atleast 3 dimensional" ?
 Similar to other files as well.